lib/one_model.ex

```diff
@@ -726,16 +722,15 @@ defmodule OneModel do
   end
 
   def paging_stats(params, fun) do
-    query_params = query_params(params)
-    offset = query_params[:offset] || 0
-
-    [{:offset, to_integer(offset || 0)}] ++ fun.()
+    offset = params |> query_params() |> Map.get(:offset, 0)
+    Keyword.put(fun.(), :offset, offset)
```

Comment:
Interesting. I had to test this. Keyword.put prepends the new value to the existing list. Efficient, but surprising to me.

Reply:
Yep, exactly! I confirmed it but looking at the source code. That's what makes it both an efficient and elegant solution.